themes/src/main/resources-community/theme/base/login/messages/messages_pt_BR.properties

```diff
@@ -151,53 +162,55 @@ role_manage-identity-providers=Gerenciar provedores de identidade
 role_manage-clients=Gerenciar clientes
 role_manage-events=Gerenciar eventos
 role_view-profile=Visualizar perfil
-role_manage-account=Gerenciar contas
-role_manage-account-links=Gerenciar v\00ED nculos de conta
+role_manage-account=Gerenciar conta
+role_manage-account-links=Gerenciar vincula\u00e7\u00f5es de conta
 role_read-token=Ler token
 role_offline-access=Acesso offline
 client_account=Conta
 client_account-console=Console da Conta
 client_security-admin-console=Console de Administra\u00E7\u00E3o de Seguran\u00E7a
-client_admin-cli=Admin CLI
+client_admin-cli=CLI de Administra\u00e7\u00e3o
 client_realm-management=Gerenciamento de Dom\u00EDnio
 client_broker=Provedor de Identidade
 
 requiredFields=Campos obrigat\u00F3rios
 
 invalidUserMessage=Nome de usu\u00E1rio ou senha inv\u00E1lida.
 invalidUsernameMessage=Nome de usu\u00E1rio inv\u00E1lido.
-invalidUsernameOrEmailMessage=Nome de usu\u00E1rio ou e-mail inv\u00E1lido.
+invalidUsernameOrEmailMessage=Nome de usu\u00E1rio ou endere\u00e7o de e-mail inv\u00E1lido.
 invalidPasswordMessage=Senha inv\u00E1lida.
-invalidEmailMessage=E-mail inv\u00E1lido.
-accountDisabledMessage=Conta desativada, contate o administrador.
-accountTemporarilyDisabledMessage=Conta temporariamente indispon\u00EDvel, contate o administrador ou tente novamente mais tarde.
+invalidEmailMessage=Endere\u00e7o de e-mail inv\u00E1lido.
+accountDisabledMessage=Conta desativada, por favor, contate um administrador.
+accountTemporarilyDisabledMessage=Conta temporariamente indispon\u00EDvel, por favor, contate um administrador ou tente novamente mais tarde.
 expiredCodeMessage=Tempo de login expirado. Por favor, fa\u00E7a login novamente.
 expiredActionMessage=A\u00E7\u00E3o expirada. Por favor, continue com o login agora.
 expiredActionTokenNoSessionMessage=A\u00E7\u00E3o expirada.
 expiredActionTokenSessionExistsMessage=A\u00E7\u00E3o expirada. Por favor, comece novamente.
 
 missingFirstNameMessage=Por favor, informe o primeiro nome.
 missingLastNameMessage=Por favor, informe o sobrenome.
-missingEmailMessage=Por favor, informe o e-mail.
+missingEmailMessage=Por favor, informe o endere\u00e7o de e-mail.
 missingUsernameMessage=Por favor, informe o nome de usu\u00E1rio.
 missingPasswordMessage=Por favor, informe a senha.
-missingTotpMessage=Por favor, informe o c\u00F3digo autenticador.
+missingTotpMessage=Por favor, informe o c\u00F3digo de uso \u00fanico.
 missingTotpDeviceNameMessage=Por favor, informe o nome do dispositivo.
 notMatchPasswordMessage=As senhas n\u00E3o coincidem.
 
 invalidPasswordExistingMessage=Senha atual inv\u00E1lida.
-invalidPasswordBlacklistedMessage=Senha inv\u00E1lida de acordo com as pol\u00EDticas estabelecidas para o dom\u00EDnio.
```

Comment:
I think the previous one is better?

Reply:
It can lead to an ambiguous error message if we're trying to be consistent with the `account` errors for blacklisted and generic messages:

```properties
# account/messages_en.properties
invalidPasswordBlacklistedMessage=Invalid password: password is blacklisted.
invalidPasswordGenericMessage=Invalid password: new password doesn''t match password policies.

# account/messages_pt_BR.properties, un-escaped
#   - Current translation:
# invalidPasswordBlacklistedMessage=Senha inválida: esta senha está na lista de exclusão.
#   - Changing the translation to correspond with the login message:
invalidPasswordBlacklistedMessage=Senha inválida: a nova senha não está de acordo com as políticas estabelecidas para o domínio.
invalidPasswordGenericMessage=Senha inválida: a nova senha não cumpre as políticas de senha.
```

An alternative to "lista de exclusão" could be "lista negra", or "lista de restrições"[[1]](https://pt.wikipedia.org/wiki/Lista_negra) to avoid potentially problematic language.

Reply:
I see. I think makes more sense to leave as is so that we don't make explicit why the password is invalid.